bug(android): build failure for 16.0-beta

```
[17:45:15][Step 2/6]   ...
[17:45:15][Step 2/6]   Errors found:
[17:45:15][Step 2/6]   
[17:45:15][Step 2/6]   C:\BuildAgent\work\78a54344b259d3bb\keyman\oem\firstvoices\android\app\src\main\java\com\firstvoices\keyboards\FVKeyboardSettingsActivity.java:166: Error: Format string 'keyboard_version' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
[17:45:15][Step 2/6]       fvVersionTextView.setText(String.format(getString(R.string.keyboard_version), version));
[17:45:15][Step 2/6]                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[17:45:15][Step 2/6]       C:\BuildAgent\work\78a54344b259d3bb\keyman\oem\firstvoices\android\app\src\main\res\values\strings.xml:10: This definition does not require arguments
```

This looks like a regression coming out of #7908.

--- a/oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVKeyboardSettingsActivity.java
+++ b/oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVKeyboardSettingsActivity.java
@@ -163,7 +163,7 @@ public final class FVKeyboardSettingsActivity extends AppCompatActivity {
     fvKeyboardTextView = (TextView) layout.findViewById(R.id.text1);
     fvKeyboardTextView.setText(getString(R.string.enable_keyboard));
     fvVersionTextView = (TextView) layout.findViewById(R.id.text2);
-    fvVersionTextView.setText(String.format(getString(R.string.keyboard_version2), version));
+    fvVersionTextView.setText(String.format(getString(R.string.fv_keyboard_version), version));
     fvKeyboardToggle = layout.findViewById(R.id.toggle);
     fvKeyboardToggle.setChecked(KeyboardController.getInstance().keyboardExists(FVShared.FVDefault_PackageID, kbId, null));
     fvKeyboardToggle.setOnClickListener(new View.OnClickListener() {

--- a/oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVKeyboardSettingsActivity.java
+++ b/oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVKeyboardSettingsActivity.java
@@ -163,7 +163,7 @@ public final class FVKeyboardSettingsActivity extends AppCompatActivity {
     fvKeyboardTextView = (TextView) layout.findViewById(R.id.text1);
     fvKeyboardTextView.setText(getString(R.string.enable_keyboard));
     fvVersionTextView = (TextView) layout.findViewById(R.id.text2);
-    fvVersionTextView.setText(String.format(getString(R.string.keyboard_version), version));
+    fvVersionTextView.setText(String.format(getString(R.string.keyboard_version2), version));
     fvKeyboardToggle = layout.findViewById(R.id.toggle);
     fvKeyboardToggle.setChecked(KeyboardController.getInstance().keyboardExists(FVShared.FVDefault_PackageID, kbId, null));
     fvKeyboardToggle.setOnClickListener(new View.OnClickListener() {

--- a/oem/firstvoices/android/app/src/main/res/values/strings.xml
+++ b/oem/firstvoices/android/app/src/main/res/values/strings.xml
@@ -7,8 +7,7 @@
     <string name="dictionaries">Dictionaries</string>
     <string name="keyboard_settings">%1$s Settings</string>
     <string name="enable_keyboard">Enable keyboard</string>
-    <string name="keyboard_version">Version</string>
-    <string name="keyboard_version2">Version %1$s</string>
+    <string name="fv_keyboard_version">Version %1$s</string>
 
     <!-- Context: KMP Package strings -->
     <string name="title_package_failed_to_install" comment="Error dialog title when package failed to install">Package %1$s failed to install</string>

--- a/oem/firstvoices/android/app/src/main/res/values/strings.xml
+++ b/oem/firstvoices/android/app/src/main/res/values/strings.xml
@@ -8,6 +8,7 @@
     <string name="keyboard_settings">%1$s Settings</string>
     <string name="enable_keyboard">Enable keyboard</string>
     <string name="keyboard_version">Version</string>
+    <string name="keyboard_version2">Version %1$s</string>
 
     <!-- Context: KMP Package strings -->
     <string name="title_package_failed_to_install" comment="Error dialog title when package failed to install">Package %1$s failed to install</string>

